Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -126,8 +126,8 @@ - (void)startWithOptions:(SentryReplayOptions *)replayOptions
     SentryOnDemandReplay *replayMaker = [[SentryOnDemandReplay alloc] initWithOutputPath:docs.path];
     replayMaker.bitRate = replayOptions.replayBitRate;
     replayMaker.cacheMaxSize
-        = (NSInteger)(shouldReplayFullSession ? replayOptions.sessionSegmentDuration
-                                              : replayOptions.errorReplayDuration);
+        = (NSInteger)(shouldReplayFullSession ? replayOptions.sessionSegmentDuration + 1
+                                              : replayOptions.errorReplayDuration + 1);
```

Comment:
What's the purpose of this change?

Reply:
This is something I noticed while working on the PR. It is just an extra room for the cache to avoid flushing frames too hastily.